train.py

```diff
@@ -163,7 +163,11 @@ def get_batch(split):
     # force these config attributes to be equal otherwise we can't even resume training
     # the rest of the attributes (e.g. dropout) can stay as desired from command line
     for k in ['n_layer', 'n_head', 'n_embd', 'block_size', 'bias', 'vocab_size', 'lora_rank', 'lora_alpha']:
-        model_args[k] = checkpoint_model_args[k]
+        try:
+            model_args[k] = checkpoint_model_args[k]
+        except KeyError as ex:
+            if k in [ 'lora_rank', 'lora_alpha' ]:
+                model_args[k] = 0 # allow resuming models that didn't include lora
```

Comment:
I'll prefer the following instead:
```suggestion
        model_args[k] = checkpoint_model_args.get(k, 0)
```

Reply:
That's cleaner, yes.  Thanks!